lib/ansible/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -783,6 +783,11 @@ def stop_vm(module, proxmox, vm, vmid, timeout, force):
     return False
 
 
+def proxmox_version(proxmox):
+    apireturn = proxmox.version.get()
+    return LooseVersion(apireturn['version'])  # < Proxmox 6
```

Comment:
```suggestion
    return LooseVersion(apireturn['version'])
```
Or is there a reason for this comment here (and not in the other module)?

Reply:
The reason is: I forgot to delete it. :sweat_smile:

Reply:
Hehe, happens ;-)